neuro_san/client/direct_agent_session_factory.py

```diff
@@ -41,11 +41,10 @@ def __init__(self):
         Constructor
         """
         manifest_restorer = RegistryManifestRestorer()
-        self.manifest_tool_registries: Dict[str, AgentToolRegistry] = manifest_restorer.restore()
-        tool_factory: ServiceToolFactoryProvider =\
-            ServiceToolFactoryProvider.get_instance()
-        for agent_name, tool_registry in self.manifest_tool_registries.items():
-            tool_factory.add_agent_tool_registry(agent_name, tool_registry)
+        self.manifest_networks: Dict[str, AgentNetwork] = manifest_restorer.restore()
+        network_storage: ServiceAgentNetworkStorage = ServiceAgentNetworkStorage.get_instance()
+        for agent_name, agent_network in self.manifest_networks.items():
+            network_storage.add_agent_network(agent_name, agent_network)
```

Comment:
Use agent_network in method/class/local_variable nomenclature for these classes instead of tool_registry/tool_factory.